traefik.sample.toml

```diff
@@ -1059,26 +1059,49 @@
 #
 # ExposedByDefault = false
 
-# Filter services with unhealthy states and health states
+# Filter services with unhealthy states and inactive states
 #
 # Optional
 # Default: false
 #
-# EnableServiceHealthFilter = false
+# EnableServiceHealthFilter = true
 
-# Endpoint to use when connecting to Rancher
+# Long poll the Rancher metadata service for changes (accurate, almost instantaneous updates)
+# NOTE: if false, the metadata service is instead polled for changes every `RefreshSeconds`
 #
-# Required
+# Optional
+# Default: true
+#
+# MetadataServiceLongPoll = false
+
+# Prefix used for accessing the Rancher metadata service
+#
+# Optional
+# Default: "/latest"
+#
+# MetadataServicePrefix = "/2016-07-29"
+
+# Source Traefik configuration from the Rancher API instead of the metadata service
+# NOTE: further requires `Endpoint`, `AccessKey`, `SecretKey`
+#
+# Optional
+# Default: false
+#
+# EnableAPI = true
+
+# Endpoint to use when connecting to the Rancher API
+#
+# Required (unless EnableMetadataService = true)
 # Endpoint = "http://rancherserver.example.com/v1"
 
-# AccessKey to use when connecting to Rancher
+# AccessKey to use when connecting to the Rancher API
 #
-# Required
+# Required (unless EnableMetadataService = true)
```

Comment:
Woops. This was a remnant from the previous iteration when metadata service was not the default.

The config is actually `EnableAPI`. I've reworded the sample to reflect this:
```
# Required (if EnableAPI = true)
... 
```